dist/Tie-File/lib/Tie/File.pm

```diff
@@ -2003,7 +2008,7 @@ sub _nodes {
   ($self->[$i], $self->_nodes($i*2), $self->_nodes($i*2+1));
 }
 
-"Cogito, ergo sum.";  # don't forget to return a true value from the file
+1;
```

Comment:
No need for fancy here.

Reply:
Please leave it in.  It's the most characteristically MJD statement in the entire codebase.

Reply:
I removed it because of a warning.

```
$>perl -cw lib/Tie/File.pm 
Useless use of a constant ("Cogito, ergo sum.") in void context at lib/Tie/File.pm line 1999.
lib/Tie/File.pm syntax OK
```